dropwizard-jersey/src/main/java/io/dropwizard/jersey/validation/JerseyViolationExceptionMapper.java

```diff
@@ -0,0 +1,29 @@
+package io.dropwizard.jersey.validation;
+
+import com.google.common.base.Function;
+import com.google.common.collect.FluentIterable;
+import com.google.common.collect.ImmutableList;
+
+import javax.validation.ConstraintViolation;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import java.util.Set;
+
+public class JerseyViolationExceptionMapper implements ExceptionMapper<JerseyViolationException> {
```

Comment:
Not to get off topic, but this should already be possible if you define your own `ExceptionMapper<ConstraintViolationException>` and register it. (You'll also need to set `registerDefaultExceptionMappers` to false). Feel free to post on the dropwizard-user channel for additional questions.